src/sass/style.scss

```diff
@@ -31,17 +31,22 @@
 	font-weight: bold;
 }
 
-.successPostTweet {
+.successPostTweet, .successReplyTweet {
```

Comment:
you could use a generic class instead of customizing it if you wanted to, this works as well